bot/cogs/games/games.py

```diff
@@ -1,4 +1,5 @@
 import random
+import aiohttp
```

Comment:
```suggestion
import aiohttp
import random
```
Maintaining the import order.